index.js

```diff
@@ -81,11 +81,18 @@ function assembleStyles() {
 		for (const styleName of Object.keys(group)) {
 			const style = group[styleName];
 
-			styles[styleName] = {
+			let tags = {
 				open: `\u001B[${style[0]}m`,
 				close: `\u001B[${style[1]}m`
 			};
 
+			styles[styleName] = (...str) => {
+				return `${tags.open}${str.join(' ')}${tags.close}`
```

Comment:
There's no point in using a template literal. There are only variables. Just use `+`.